syz-ci/syz-ci.go

```diff
@@ -103,11 +103,12 @@ type ManagerConfig struct {
 	DashboardKey    string `json:"dashboard_key"`
 	Repo            string `json:"repo"`
 	// Short name of the repo (e.g. "linux-next"), used only for reporting.
-	RepoAlias    string `json:"repo_alias"`
-	Branch       string `json:"branch"` // Defaults to "master".
-	Compiler     string `json:"compiler"`
-	Userspace    string `json:"userspace"`
-	KernelConfig string `json:"kernel_config"`
+	RepoAlias            string `json:"repo_alias"`
+	Branch               string `json:"branch"` // Defaults to "master".
+	Compiler             string `json:"compiler"`
+	Userspace            string `json:"userspace"`
+	KernelConfig         string `json:"kernel_config"`
+	KernelBaselineConfig string `json:"kernel_baseline_config"`
```

Comment:
I know I am not always lavish re comments, but it does not stop me from asking others :)
Please add: `// Baseline config for bisection, see pkg/bisect.KernelConfig.BaselineConfig.`

Reply:
done